std/random.d

```diff
@@ -623,17 +623,20 @@ Parameters for the generator.
     void seed(T)(T range) if(isInputRange!T && is(Unqual!(ElementType!T) == UIntType))
     {
         size_t j;
-        for(j = 0; j < n && !range.empty; ++j, range.popFront())
+        for (j = 0; j < n && !range.empty; ++j, range.popFront())
         {
             mt[j] = range.front;
         }
 
         mti = n;
-        if(range.empty && j < n)
+        if (range.empty && j < n)
         {
-            import std.format : format;
-            throw new Exception(format("MersenneTwisterEngine.seed: Input range didn't provide enough"~
-                " elements: Need %s elemnets.", n));
+            import core.internal.string;
+
+            UnsignedStringBuf buf = void;
+            string s = "MersenneTwisterEngine.seed: Input range didn't provide enough elements: Need ";
+            s ~= unsignedToTempString(n, buf, 10) ~ " elements.";
```

Comment:
It's a workaround for: https://issues.dlang.org/show_bug.cgi?id=14752

I'm not too concerned about it here, because this code should pretty much never be executed, and arguably should be a fatal error anyway (not a recoverable Exception).